snowflake/build_and_publish_rpm.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -ex
+#
+# Copyright (c) 2019 Snowflake Computing Inc. All right reserved.
+#
+#
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source $THIS_DIR/version.sh
+TARGET_YUM_REPO=$1
+
+if [[ -z $TARGET_YUM_REPO ]]; then
+  echo "ERROR: {TARGET_YUM_REPO} not specified"
+  exit 1
+fi
+
+
+$THIS_DIR/../platform/rpm/mkrpm.sh firejail $FIREJAIL_SF_VERSION \
+    "--disable-userns --disable-contrib-install --disable-file-transfer --disable-x11 --disable-firetunnel"
```

Comment:
Why disable-userns?

Reply:
Default RPM build is disabling this. Do we need this?